sandbox/Foundation/src/Foundation/Startup.cs

```diff
@@ -91,7 +96,9 @@ public void ConfigureServices(IServiceCollection services)
             services.AddDetection();
             services.AddTinyMceConfiguration();
 
-            services.AddSitemaps();
+            services.AddSingleton<SitemapUriParameterAugmenterService>();
```

Comment:
Consumer apps should not explicitly register augmenter service. See the comment, above for more info.